projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3913,6 +3913,11 @@ public function upload_handler( $update_media_item = false ) {
 			return new WP_Error( 'unknown_token', 'Unknown Jetpack token', 403 );
 		}
 
+		$upload_blocked = apply_filters( 'jetpack_upload_handler_block', false, $_FILES );
```

Comment:
Can you add a docstring for this filter? Something similar to this format would be great:

https://github.com/Automattic/jetpack/blob/05f9ad44f1725e736d38bdf080d02b641d850b8f/projects/plugins/jetpack/class.jetpack.php#L4870-L4878